apps/artist/routes.coffee

```diff
@@ -24,10 +24,8 @@ module.exports.index = (req, res, next) ->
   Q.all([
     artist.fetch(cache: true)
     artist.maybeFetchAndSetFeaturedBio()
-    artist.fetchArtworks(data: {size: 50})
```

Comment:
@1aurabrown I believe this `fetchArtworks` was happening not to actually render the artworks but as part of [Criteo support](https://github.com/artsy/microgravity/pull/93).